src/components/PopupWithForm.js

```diff
@@ -18,21 +19,34 @@ class PopupWithForm extends Popup {
 
     this._inputList.forEach((input) => {
       inputValues[input.name] = input.value;
+      console.log(input.value);
     });
 
+    console.log(this._inputList);
+    console.log(inputValues);
+
     return inputValues;
   }
 
+  open(){
+    super.open();
+    this.saveButton.textContent = "Save";
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.